File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/drug/DRG0102U00GrdDetailInfo.java
package nta.med.data.model.ihis.drug;

public class DRG0102U00GrdDetailInfo {
	private String codeType;
    private String code;
    private String codeName;
    private String codeValue;
	public DRG0102U00GrdDetailInfo(String codeType, String code,
			String codeName, String codeValue) {
		super();
		this.codeType = codeType;
		this.code = code;
		this.codeName = codeName;
		this.codeValue = codeValue;
	}
	public String getCodeType() {
		return codeType;
	}
	public void setCodeType(String codeType) {
		this.codeType = codeType;
	}
	public String getCode() {
		return code;
	}
	public void setCode(String code) {
		this.code = code;
	}
	public String getCodeName() {
		return codeName;
	}
	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}
	public String getCodeValue() {
		return codeValue;
	}
	public void setCodeValue(String codeValue) {
		this.codeValue = codeValue;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Drgs/DRG0102U00GrdDetailInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Drgs
{
    [Serializable]
	public class DRG0102U00GrdDetailInfo
	{
		private String _codeType;
		private String _code;
		private String _codeName;
		private String _codeValue;
		private String _rowState;

		public String CodeType
		{
			get { return this._codeType; }
			set { this._codeType = value; }
		}

		public String Code
		{
			get { return this._code; }
			set { this._code = value; }
		}

		public String CodeName
		{
			get { return this._codeName; }
			set { this._codeName = value; }
		}

		public String CodeValue
		{
			get { return this._codeValue; }
			set { this._codeValue = value; }
		}

		public String RowState
		{
			get { return this._rowState; }
			set { this._rowState = value; }
		}

		public DRG0102U00GrdDetailInfo() { }

		public DRG0102U00GrdDetailInfo(String codeType, String code, String codeName, String codeValue, String rowState)
		{
			this._codeType = codeType;
			this._code = code;
			this._codeName = codeName;
			this._codeValue = codeValue;
			this._rowState = rowState;
		}

	}
}